src/swagger/specGenerator3.ts

```diff
@@ -290,10 +290,6 @@ export class SpecGenerator3 extends SpecGenerator {
       parameter.schema.format = this.throwIfNotDataFormat(parameterType.format);
     }
 
-    if (parameter.in === 'query' && parameterType.type === 'array') {
-      (parameter as Swagger.QueryParameter).collectionFormat = 'multi';
```

Comment:
I’m really surprised that this didn’t break any unit tests.

Reply:
Same, but then that would've meant that someone explicitly expected wrong behaviour.

Reply:
I only noticed because it broke our build that validates the spec and errors it it has any issues including addfitional keys.

Reply:
WoW